hash/carbeena.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+<!--
+license:CC0-1.0
+
+This cartridge is inserted into the mainboard slot, enclosed inside the console.
+-->
+<softwarelist name="carbeena" description="Car Beena cartridge">
+
+	<software name="carbeena" supported="yes">
+		<description>Car Beena Tentou-you Demo 1</description>
+		<year>2009</year>
+		<publisher>Sega Toys</publisher>
+		<notes>Info from header</notes>
+		<info name="alt_title" value="カービーナてんとうデモよう1"/>
```

Comment:
This is written as "Tentou Demo-you", but I guess "Tentou-you Demo" is more common in latin script?